### ISSUE TITLE
Add imapsync container

### ISSUE THREAD

**Issue by hpannenb** [bashclub/zamba-lxc-toolbox#63]
Under some circumstances it might be useful to have a dedicated imapsync LXC available.

Here a reference to a procedure installing an additional (apache) web based imapsync:
https://www.bachmann-lan.de/imapsync-unter-debian-11-installieren/

**Comment by fbartels** [comment]
Nowadays I really like running imapsync as a docker container:

```
docker run --rm --network host gilleslamiral/imapsync imapsync --no-modulesversion --noreleasecheck --host1 xxx --user1 xxx  --password1 'xxx' --host2 yyy --user2 yyy --password2 'yyy' --automap 
```

**Comment by udo1toni** [comment]
As a matter of fact, proxmox is more targeted to lxc. Moreover, bashclub is kind of biased against docker, for reasons... ;)

**Comment by fbartels** [comment]
> lxc 

These are different tools, for different purposes ;)

**Comment by udo1toni** [comment]
Yes fore sure, but there is no docker on proxmox, so you would have to build a container with docker (which is easy enough), and as lxc is another containerized system (while going another way as docker), it seems a bit odd to do it that way.
